kpops/components/streams_bootstrap_v3/__init__.py

```diff
@@ -0,0 +1,6 @@
+from kpops.components.common.streams_bootstrap import StreamsBootstrap
```

Comment:
shouldn't it export `StreamBootstrapV3`?

Reply:
Yes, corrected